### PR TITLE
StatsFrame Name Bugfix Level

### PR DESCRIPTION
**Pull request by michaelmckinsey1**
# Summary
When the statsframe columns are multiindex, the name column should reside at the same level as in the PerfData.

### CODE REVIEW

File: thicket/tests/test_thicket.py [hunk 1]
```diff
@@ -71,7 +71,18 @@ def test_sync_nodes(example_cali):
     assert helpers._are_synced(th.graph, th.dataframe)
 
 
-def test_aggregated_statistics_table(example_cali):
```
Comment (slabasan) [inline thicket/tests/test_thicket.py:74]: Why the change of the test name here?

Comment (michaelmckinsey1) [reply thicket/tests/test_thicket.py:74]: The pattern we follow with all the other test functions is to use the name of the attribute/function name. I think someone accidentally changed this function in a find and replace since it used to be `test_statsframe`

**Review by slabasan — APPROVED** [review]
